### PR TITLE
Serbian translation (sr and sr@latin).

### PR DESCRIPTION
**Pull request by Faenriis**
Changes made in this pull request: 
Added Serbian translation (cyrillic and latin script).

Please merge, thank you.

### CODE REVIEW

File: po/sr.po [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
+"Language: sr\n"
+
+#: /home/felipe/Code/com.github.philip-scott.spice-up/po/../src/Services/GamepadSlideController.vala:66
+msgid "Controller found"
+msgstr "Нађен контролер"
+
+#: /home/felipe/Code/com.github.philip-scott.spice-up/po/../src/Services/GamepadSlideController.vala:67
+msgid "Set up…"
+msgstr "Подеси ..."
```
Comment (Philip-Scott) [inline po/sr.po]: Please use ellipsis here, and everywhere else with 3 dots :) 
`…`

File: po/sr.po [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+#: /home/felipe/Code/com.github.philip-scott.spice-up/po/../src/Services/GamepadSlideController.vala:188
+#: /home/felipe/Code/com.github.philip-scott.spice-up/po/../src/Services/GamepadSlideController.vala:200
+#: /home/felipe/Code/com.github.philip-scott.spice-up/po/../src/Services/GamepadSlideController.vala:203
+#, c-format
+msgid "Button %d"
+msgstr "Дугме %d"
+
+#: /home/felipe/Code/com.github.philip-scott.spice-up/po/../src/Services/GamepadSlideController.vala:210
+msgid "..."
+msgstr "..."
```
Comment (Philip-Scott) [inline po/sr.po]: except here :sweat_smile:

File: po/sr.po [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+msgid "Start Presentation"
+msgstr "Покрени презентацију"
+
+#: /home/felipe/Code/com.github.philip-scott.spice-up/po/../src/Widgets/Templates/TemplateItem.vala:34
+msgid "Create new presentation"
+msgstr "Направи нову презентацију"
+
+#: /home/felipe/Code/com.github.philip-scott.spice-up/po/../src/Widgets/ColorButton.vala:98
+msgid "Custom Color"
+msgstr "Произвољна боја"
```
Comment (Philip-Scott) [inline po/sr.po]: Google translate is changing this to "Random color". Can you confirm it's correct? :)

File: po/sr.po [hunk 4]
```diff
@@ -0,0 +1,385 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (Philip-Scott) [inline po/sr.po]: Please also fill out the copyright :) 

thanks

File: po/sr.po (hunk 3 above)
Comment (Faenriis) [reply po/sr.po]: Yes, Google Translate often fails to see the small subtle differences in tone of Slavic languages, like "I'll choose randomly for you" and "Choose whatever you feel like" (there is a bit of randomness in both, difference is who makes the choice). I changed it to "prilagođena" to be more specific, though it'll probably say it means "adjusted". :)

**Comment by Philip-Scott** [comment]
Thank you sir!

**Pull request merged**